README.md

```diff
@@ -19,10 +19,10 @@ The goal of this specification is to enable the creation of interoperable tools
 
 ## Overview
 
-This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
-At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more [filesystem layer changeset](layer.md) archives that will be unpacked to make up the final runnable filesystem.
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md) and may with a [manifest list](manifest-list.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
+At a high level the manifest list points to specific image manifests for one or more platform and the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more [filesystem layer changeset](layer.md) archives that will be unpacked to make up the final runnable filesystem.
```

Comment:
There's no need to wedge both manifest lists and manifests into the same sentence.  Can we have an “At a high level, …” one-liner just for manifest lists followed by a separate line for manifests and layers?

I'm also tempted to move the manifest-list line below the manifest line, since fat, multi-platform images are easier to understand if you already know what a manifest is.  Manifests rely on filesystems and configurations, but the reader is more likely to be able to guess at those than they are to guess the meaning of “manifest”.